tests/installation/bootloader_start.pm

```diff
@@ -32,46 +32,62 @@ BEGIN {
     unshift @INC, dirname(__FILE__) . '/../boot';
 }
 use boot_from_pxe;
+use uefi_bootmenu;
 
 sub run {
     my $self = shift;
     if (uses_qa_net_hardware() || get_var("PXEBOOT")) {
+        record_info('Starting boot_from_pxe');
```

Comment:
I'd go for 
```suggestion
        record_info('boot_from_pxe');
```

and the same for the rest of the bootloaders, ast the title in the record info box is smaller

Reply:
done